contribution/translation/type.php

```diff
@@ -154,10 +154,11 @@ public function translation_validate(\titania_contribution $contrib, \titania_re
 
 		$version = $revision->phpbb_versions[0];
 
-		if ($version['phpbb_version_branch'] != 30)
+		if ($version['phpbb_version_branch'] != 32)
```

Comment:
Is there a constant defined somewhere that we can use here?

Reply:
I can't find anything. We could take the highest version number from https://github.com/phpbb/customisation-db/blob/3.2.x/config/config.php#L158 maybe?

Reply:
Sounds good!

If we want to plan ahead, maybe we could add a new config entry in that file. At some point we might want to be able to allow language pack submissions for 3.2.x and 3.3.x at the same time.